core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.html

```diff
@@ -73,6 +73,12 @@ <h3 class="subtopic-card-heading">Review Material</h3>
                 </em>
               </span>
             </form>
+            <span class="form-text error-msg" *ngIf="errorMsg">
+              <em>{{ errorMsg }}</em>
+            </span>
+            <span class="form-text error-message" *ngIf="editableUrlFragment?.length === 25">
```

Comment:
In the subtopic editor, we have two input fields that have a maxLength limit. The field for the subtopic title displays a message when the maximum character limit is reached. However, the field for the subtopic URL fragment did not display this message, so I have added it for consistency. The limit of 25 characters is defined in the [constants.ts](https://github.com/oppia/oppia/blob/develop/assets/constants.ts#L6221) file.